docker-compose.yml

```diff
@@ -105,3 +105,20 @@ services:
         retries: 6
     ports:
       - "1434:1433"
+
+  init-service:
+      build:
+        context: .
+        dockerfile: init-service/Dockerfile
```

Comment:
We can take it for a vote . but personally I prefer 'init process' as its not really a 'service' in the wider term.
process also have a connotation of a one time thing.
but I am also ok with leaving it 'service' to be aligned with the others. wdyt.
@tamirkamara

Reply:
vote in progress on teams poll.